src/Polly.Core/Retry/OnRetryArguments.cs

```diff
@@ -14,8 +14,8 @@ namespace Polly.Retry;
     /// <summary>
     /// Initializes a new instance of the <see cref="OnRetryArguments{TResult}"/> struct.
     /// </summary>
-    /// <param name="outcome">The context in which the resilience operation or event occurred.</param>
-    /// <param name="context">The outcome of the resilience operation or event.</param>
+    /// <param name="outcome">The outcome of the resilience operation or event.</param>
+    /// <param name="context">The context in which the resilience operation or event occurred.</param>
```

Comment:
Based on the order of the declaration of the arguments, the fix should be:

```suggestion
    /// <param name="context">The context in which the resilience operation or event occurred.</param>
    /// <param name="outcome">The outcome of the resilience operation or event.</param>
```

Reply:
Fixed, could you please review again